runtime/Cargo.toml

```diff
@@ -33,45 +33,50 @@ url = "2"
 cfg-if = "1.0"
 prometheus = { version = "0.12.0", features = ["process"] }
 lazy_static = "1.4.0"
+base58 = { version = "0.2.0" }
+blake2 = { version = "0.10.4", default-features = false }
+scale-decode = { version = "0.7.0",  features = ["derive"] }
+scale-encode = { version = "0.3.0",  features = ["derive"] }
 
 # Substrate dependencies
 sp-arithmetic = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40", default-features = false }
 # needs to agree with subxt's version
 sp-keyring = { version = "7.0.0", default_features = false }
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
 frame-support = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.40" }
 
 # Subxt dependencies
-subxt = "0.25.0"
+subxt = { version = "0.29.0", default_features = false,  features = ["jsonrpsee-ws"] }
 subxt-client = { path = "./client", optional = true }
-jsonrpsee = { version = "0.16.2", features = ["async-client", "client-ws-transport", "macros", "jsonrpsee-types", "client", "jsonrpsee-ws-client", "jsonrpsee-client-transport"] }
+jsonrpsee = { version = "0.16", features = ["async-client", "client-ws-transport", "macros", "jsonrpsee-types", "client", "jsonrpsee-ws-client", "jsonrpsee-client-transport"] }
 
 bitcoin = { path = "../bitcoin" }
 
 # Dependencies for the testing utils for integration tests
 tempdir = { version = "0.3.7", optional = true }
-interbtc = { package = "interbtc-parachain", git = "https://github.com/interlay/interbtc", rev = "7a2f5ee35d93feb0529a308fe87b42674bfca8e9", optional = true }
-interbtc-runtime = { package = "testnet-kintsugi-runtime-parachain", git = "https://github.com/interlay/interbtc", rev = "7a2f5ee35d93feb0529a308fe87b42674bfca8e9", optional = true }
+interbtc = { package = "interbtc-parachain", git = "https://github.com/interlay/interbtc", branch = "nakul/feat_add_trait", optional = true }
+interbtc-runtime = { package = "testnet-kintsugi-runtime-parachain", git = "https://github.com/interlay/interbtc", branch = "nakul/feat_add_trait", optional = true }
 rand = { version = "0.7", optional = true }
 
 [dependencies.primitives]
 git = "https://github.com/interlay/interbtc"
-rev = "7a2f5ee35d93feb0529a308fe87b42674bfca8e9"
+branch = "nakul/feat_add_trait"
```

Comment:
Yes, there are changes on the node side, but there's a problem here `nakul/feat_add_trait` is branched out from commit id `7a2f5ee35d93feb0529a308fe87b42674bfca8e9` and not `master` since, the `master` doesn't contain testnet runtimes. The best solution is to merge PR #476 first, then `nakul/feat_add_trait` can be merged to master. After this the dependencies can point out to the master commit.

Reply:
Okay, please already open a PR to parachain master with the suggested changes.